src/utils/arweave.ts

```diff
@@ -188,20 +189,31 @@ export const getEpochDataFromGqlFallback = async ({
       continue;
     }
     for (const message of messageResult?.Messages ?? []) {
-      const data = JSON.parse(message.Data);
```

Comment:
The real fix that we need here is that we have to filter out messages that don't contain Data (e.g. patch messages).